rs-libc/src/lib.rs

```diff
@@ -5,3 +5,13 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 pub mod alloc;
+
+extern crate zeroize;
+use zeroize::Zeroize;
+
+#[no_mangle]
+/// Rust version of [`explicit_bzero`], implemented by using [`zeroize::Zeroize`](https://docs.rs/zeroize/latest/zeroize/trait.Zeroize.html)
+pub unsafe extern "C" fn explicit_bzero(buf: *mut std::ffi::c_void, len: alloc::size_t)  {
+    let buffer = core::slice::from_raw_parts_mut(buf as *mut std::ffi::c_char, len);
+    buffer.zeroize();
```

Comment:
I'm also sure it works, but can we still add a test?

Reply:
tests added